sites/svelte.dev/src/routes/_components/Demo.svelte

```diff
@@ -28,7 +28,7 @@
 		},
 	];
 
-	let selected = examples[0];
+	$: selected = examples[0];
```

Comment:
why this change? `examples[0]` is only the initial value, it's updated in an event handler. even if it 'works' because of how svelte's reactivity works, i'd argue it's a bit confusing

```suggestion
	let selected = examples[0];
```

Reply:
It's left over from some debugging. Resolved it